pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -162,6 +162,24 @@ func (c *Controller) clusterBackupSync(ctx context.Context, interval time.Durati
 	return nil
 }
 
+type FilterFunc = func(backup *v3.EtcdBackup) bool
+
+// filterBackups will test each backup within backups to determine if the filter validates (i.e. returns true) for the
+// given backup. If any filter is false, the backup is considered invalid.
+func filterBackups(backups []*v3.EtcdBackup, filter ...FilterFunc) []*v3.EtcdBackup {
+	ret := make([]*v3.EtcdBackup, 0)
+OUTER:
+	for _, backup := range backups {
+		for _, f := range filter {
```

Comment:
Nit: change f to a non single char variable for readability

Reply:
`f` is quite a common name for a "function" variable so I think it's fine to keep it as `f`.